content/codepipeline/githubcredentials.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "GitHub Access Token"
+date: 2018-10-087T08:30:11-07:00
+weight: 13
+draft: true
+---
+
+In order for CodePipeline to receive callbacks from GitHub, we need to generate a personal access token.
```

Comment:
Can we re-use the flow here: https://ecsworkshop.com/prerequisites/token/ 

Rather than maintaining all of these screenshots?

Reply:
For sure :)